src/frontend/src/components/ImportanceSliders.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+
+import Slider from "./base/Slider/Slider";
+
+const Factor = {
+    Commute: "commute",
+    School: "school",
+    Safety: "safety",
+};
+
+type FactorKeys = (typeof Factor)[keyof typeof Factor];
+
+type FactorType = { [key: FactorKeys]: number };
+
+const importance: FactorType = {
+    commute: 1,
+    school: 2,
+    safety: 3,
+};
+
+const ImportanceSliders = () => {
+    const [factor, setFactor] = useState<FactorType>({ ...importance });
+
+    return (
+        <div className="flex flex-col gap-8">
+            <Slider
+                step={1}
+                minValue={1}
+                maxValue={4}
+                label="Commute time"
+                value={factor.commute}
+                onChange={(value: number | number[]) => {
+                    if (Array.isArray(value)) return;
```

Comment:
```suggestion
```
Do we need this line in the `onChange`? This was confusing to see at first. I see how the type is defined as possibly an array in the SliderProps, but we're not implementing range sliders so in theory it should always just return type `number`. What do you think about taking it out for our own simplicity?